code/game/objects/items/mop.dm

```diff
@@ -53,7 +53,9 @@
 
 	if(T)
 		user.visible_message("<span class='notice'>[user] begins to clean \the [T] with [src].</span>", "<span class='notice'>You begin to clean \the [T] with [src]...</span>")
-		var/clean_speedies = user.mind.get_skill_modifier(/datum/skill/cleaning, SKILL_SPEED_MODIFIER)
+		var/clean_speedies = 1
+		if(user?.mind)
```

Comment:
Do we need to check for user here?

Reply:
I can't think of a scenario where the user would be null. I think even telekinesis has a user. I'll test it before my next commit and remove it if I can't find a way to have a null user.